src/celeritas/io/NeutronXsReader.hh

```diff
@@ -21,7 +21,20 @@ namespace celeritas
 {
 //---------------------------------------------------------------------------//
 /*!
- * Load the neutron elastic cross section (G4PARTICLEXSDATA/neutron/elZ) data.
+ * Types of microscopic cross sections in G4PARTICLEXSDATA/neutron data.
+ */
+enum class NeutronXsType
+{
+    cap,  //!< Capture cross section
+    el,  //!< Elastic cross section
+    inel,  //!< Inelastic cross section
```

Comment:
Actually, these strings are used to read external Geant4 data which are named specifically like them.  So, it would be better to keep them as they are since there is an option to read them directly from the Gant4 data directory/files.

Reply:
Makes sense. Thanks!